applications/GeoMechanicsApplication/custom_conditions/micro_climate_constants.h

```diff
@@ -0,0 +1,34 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#pragma once
+
+namespace Kratos::MicroClimateConstants
+{
+
+constexpr static auto AirDensity = 1.18;
+constexpr static auto AirHeatCapacity = 1004.67;
+constexpr static auto RoughnessLayerResistance = 30.0;
+constexpr static auto LatentEvaporationHeat = 2.45e6;
+constexpr static auto WaterDensity = 1e3;
```

Comment:
Redefinition, this already exists in our material input and does vary there. Furthermore it dictates what the system of units should be.

Reply:
Good catch! I removed it from here and used the DENSITY_WATER parameter from the material parameters in the condition